packages/chat-client/.stylelintignore

```diff
@@ -0,0 +1,8 @@
+.history
```

Comment:
`.stylelintignore` can be removed, this was an artifact from the TS lib template I used originally to set up the chat client MVP repo

Reply:
Also the `.stylelintrc`